app/src/main/scala/org/alephium/explorer/persistence/model/TransactionEntity.scala

```diff
@@ -27,6 +27,6 @@ final case class TransactionEntity(
     chainTo: GroupIndex,
     gasAmount: Int,
     gasPrice: U256,
-    index: Int,
+    order: Int,
```

Comment:
Should we avoid the order here as well?

Reply:
no in the scala entity it's all fine and cleaner, rather than repeating `Transaction.tx_order`, `Input.input_order` etc